stdlib/public/core/RangeReplaceableCollection.swift.gyb

```diff
@@ -1142,11 +1142,11 @@ extension RangeReplaceableCollection
 public func + <C : RangeReplaceableCollection, S : Sequence>(lhs: C, rhs: S) -> C
   where S.Iterator.Element == C.Iterator.Element {
 
-  var lhs = lhs
```

Comment:
I am uncomfortable with the idea that generic algorithms should have to explicitly call `copy()` to avoid reference semantics; it's a defensive copy that effectively undermines value semantics, and in similar circumstances (no other references) we wouldn't even want it to happen.  We need—and have needed, since the earliest days of Swift—a comprehensive strategy for dealing with the value/reference issue. I would prefer to leave this code alone (complete with slightly worrisome comment) until such a time as we can solve the larger problem in the language or at least the library design.